bump_dictionary/cli.py

```diff
@@ -0,0 +1,113 @@
+import json
+from pathlib import Path
+
+import typer
+from typing_extensions import Annotated
+
+from . import utils
+from .logger import VerbosityLevel, configure_logger, log_error, logger
+from .models import latest_dictionary_model, legacy_dictionary_model
+
+bump_dictionary = typer.Typer(
+    help="Bump Neurobagel data dictionaries to the latest version of the data dictionary schema.",
+    context_settings={"help_option_names": ["-h", "--help"]},
+    rich_markup_mode="rich",
+)
+
+
+@bump_dictionary.command()
+def main(
+    data_dictionary: Annotated[
+        Path,
+        typer.Argument(help="Path to a Neurobagel data dictionary JSON file."),
```

Comment:
```suggestion
        typer.Argument(help="Path to a previously valid Neurobagel data dictionary JSON file."),
```
or something like that - the to be updated file